### PR TITLE
fix chrisop to match repo

### PR DESCRIPTION
**Pull request by noamzbr**
Fix chrisop model to match 
https://github.com/cchinchristopherj/Right-Whale-Convolutional-Neural-Network/blob/master/training.py

### CODE REVIEW

File: src/conf/optim/christoph.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+optim:
+  epochs: 1000
+  optimizer:
+    _target_: torch.optim.SGD
+    lr: 0.1
+    weight_decay: 0.005
+    nesterov: False
+  scheduler:
+    _target_: torch.optim.lr_scheduler.ExponentialLR
+    gamma: 1
```
Comment (mosheman5) [inline src/conf/optim/christoph.yaml:11]: do we need a scheduler if gamma is 1?

**Review by mosheman5 — APPROVED** [review]
lgtm
